worlds/alttp/EntranceRandomizer.py

```diff
@@ -212,9 +212,7 @@ def defval(value):
                              Alternatively, can be a ALttP Rom patched with a Link
                              sprite that will be extracted.
                              ''')
-    parser.add_argument('--gui', help='Launch the GUI', action='store_true')
 
-    parser.add_argument('--enemizercli', default=defval('EnemizerCLI/EnemizerCLI.Core'))
     parser.add_argument('--shufflebosses', default=defval('none'), choices=['none', 'basic', 'normal', 'chaos',
```

Comment:
Are those other arguments ever used / is this executable ever called?

Reply:
They're still hooked into the Generate in an awkward way to build the namespace LttP expects.